co2calculator/api/emission.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Emission class"""
+
+from dataclasses import dataclass
+from pydantic import BaseModel
+
+
+@dataclass
+class Emissions:
+    """Class for storing information on emissions"""
+
+    co2e: float
+    distance: float
```

Comment:
Can you elaborate why the `Emissions` class needs a distance property? It seems unnecessary for two reasons: 

- The object already has the co2e calculated.
- I assume we reuse the Emissions class for Energy calculations as well. There, it would not make sense to store a distance attribute.

Reply:
True! We could also make it optional and add an optional "consumption" attribute as well. Then we could use it for both. I would like to keep it attached to the emission calculation results, since it is a good meta information which the user might want to retrieve in a later analysis. What do you think?

Reply:
Or we create child classes

Reply:
I am not quite sure if there is a need to keep it as a meta information because the user actually needs to provide the distance information in the first place for now (I agree that this might change once we implement/add the ORS). But we can keep it for now but would need to create child classes because it would be unclean to also use it for the energy emissions.